src/main/resources/jenkins/plugins/extracolumns/Messages.properties

```diff
@@ -44,3 +44,5 @@ BuildParametersColumn.DisplayName=Build Parameters
 LastJobConfigurationModificationColumn.DisplayName=Last Configuration Modification
 SlaveOrLabelColumn.DisplayName=Slave Allocation
 UserNameColumn.DisplayName=User Name
+CronTriggerColumn.DisplayName=Periodic Build Trigger
+CronTriggerColumn.ToolTipFormat=Would last have run at {0}; would next run at {1}.
```

Comment:
I agree too! But my intention was to make it the same as a similar message in TimerTrigger which is shown on the job ui page. That way, it will save us the effort to localize it in different languages.